File: Chronos.Domain/Entities/Estimate/Estimate.cs
using System;
using System.Collections.Generic;

namespace Chronos.Domain.Entities.Estimate
{
    public class Estimate
    {
        public int Id { get; set; }
        public int RequestOfWorkId { get; set; }
        public Version Version { get; set; }
        public int EstimateTemplateId { get; set; }
        public Uri FeatureDefinitionDocumentUrl { get; set; }
        public Uri DesignDocumentUrl { get; set; }
        public double GrandTotal { get; set; }
        public List<EstimateItem> EstimateItems{ get; set; }
    }
}

File: Chronos.App/ClientApp/src/app/models/estimate/estimate.ts
export class Estimate {
    estimateId = 0;
    version = '';
    featureDefinitionDocumentUrl = '';
    designDocumentUrl = '';
    grandTotal = 0;

    constructor(obj?: Partial<Estimate>) {
        if (obj)
            Object.assign(this, obj);
    }
}
